### PR TITLE
Use new docs reference structure

### PR DESCRIPTION
**Pull request by elenatorro**

### CODE REVIEW

File: cartoframes/data/__init__.py [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
+
 from .dataset.dataset import Dataset
 from .dataset.registry.strategies_registry import StrategiesRegistry
+from .clients.sql_client import SQLClient
 
 __all__ = [
     'Dataset',
-    'StrategiesRegistry'
+    'StrategiesRegistry',
+    'SQLClient'
```
Comment (Jesus89) [inline cartoframes/data/__init__.py]: Why is this exposed in `cartoframes.data` now (insdead of `cartoframes.data.clients`)?

File: cartoframes/data/dataset/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from __future__ import absolute_import
```
Comment (Jesus89) [inline cartoframes/data/dataset/__init__.py]: I think this is not required

File: cartoframes/data/dataset/registry/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+from __future__ import absolute_import
```
Comment (Jesus89) [inline cartoframes/data/dataset/registry/__init__.py]: I think we can remove this if we are not using and import in the file

File: cartoframes/viz/widgets/__init__.py [hunk 4]
```diff
@@ -10,23 +10,18 @@
...
-
-
-def _inspect(widget):
-    import inspect
-    lines = inspect.getsource(widget)
-    print(lines)
-
 
 __all__ = [
-    'Widget',
```
Comment (Jesus89) [inline cartoframes/viz/widgets/__init__.py]: Why are the main Widget classes being removed from the API?

File: cartoframes/data/dataset/registry/__init__.py (hunk 3 above)
Comment (elenatorro) [reply cartoframes/data/dataset/registry/__init__.py]: Honestly I'm not 100% sure of that cause I think Sphinx needs it to use `automodule` (for Pyton 2.7), but I'll test it

File: cartoframes/data/__init__.py (hunk 1 above)
Comment (elenatorro) [reply cartoframes/data/__init__.py]: We should be consistent with the current API, so let's move it back to data.clients

File: cartoframes/viz/widgets/__init__.py (hunk 4 above)
Comment (elenatorro) [reply cartoframes/viz/widgets/__init__.py]: They're in the API but they were duplicated. They're not in `widgets` namespace (that's for built-in widgets)

**Pull request merged**